src/itoolkit/transport/ssh.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+"""
+XMLSERVICE ssh call (ssh job, often in QUSRWRK)
+
+License:
+  BSD (LICENSE)
+  -- or --
+  http://yips.idevcloud.com/wiki/index.php/XMLService/LicenseXMLService
+
+  Example:
+    from itoolkit.transport import SshTransport
+    import paramiko
+    ssh = paramiko.SSHClient()
+    ssh.set_missing_host_key_policy(paramiko.WarningPolicy()) # configure paramiko separately or this is not secure
+    ssh.connect(host, username="linux", password="linux1")
+    itransport = SshTransport(ssh)
+
+Notes:
+  1) Always uses XMLSERVICE shipped in the QXMLSERV library
+  2) does not disconnect the given SSHClient object when finished
+
+"""
+import os
+
+try:
+    import paramiko
+except ImportError:
+    raise Exception("SSH library paramiko must be installed")
+    
+
+__all__ = [
+    'SshTransport'
+]
+
+class SshTransport(object):
+    """
+    Transport XMLSERVICE calls over SSH connection.
+
+    Args:
+      sshclient   (paramiko.SSHClient): A connected and authenticated SSHClient object
+
+    Example:
+      from itoolkit.transport import SshTransport
+      import paramiko
+      ssh = paramiko.SSHClient()
+      ssh.set_missing_host_key_policy(paramiko.WarningPolicy()) # configure paramiko separately or this is not secure
+      ssh.connect(host, username="linux", password="linux1")
+      itransport = SshTransport(ssh)
+
+    Returns:
+       (obj)
+    """
+
+    def __init__(self, sshclient=None):
+        #TODO: allow for SSH client to be materialized from IBM Cloud deployments
+        if not isinstance(sshclient, paramiko.SSHClient):
+          raise Exception("An instance of paramiko.SSHClient is required")
```

Comment:
All we should care about is that it provides the correct interface. So we should check this instead:

```python
hasattr(sshclient, 'exec_command')
```